backend/src/data_access_layer/map.json

```diff
@@ -0,0 +1,3 @@
+{
+    "id": "map_1"
+}
```

Comment:
Make sure to ignore this file in nodemon or else running while developing will constantly restart the server

Reply:
Make sure to also gitignore this file as well